src/sections/General/Footer/index.js

```diff
@@ -7,8 +7,75 @@ import Button from "../../../reusecore/Button";
 import FooterWrapper from "./footer.style";
 import bubblesElement from "./images/bubbles-element.svg";
 
-const Footer = () => {
+const Footer = ({ location }) => {
   var currentYear = new Date().getFullYear();
+
```

Comment:
Hi @8rxn,

There is a small outside issue that will probably need to be addressed by your code.

When your code goes into a live production build, the pathnames will end in `.html`. 

This is related to code in a PR addressing a flickering trailing slash, so you should add code to remove that `.html` (even though you will not see it in local builds).

here is an example from a simulated live production build:
![image](https://github.com/layer5io/layer5/assets/90356410/6c31d43c-9e34-4547-8519-96082cab45c9)

Let me know if you have any questions about this.

Cheers,
Randy

Reply:
This seems like a good first issue for someone.
Should I raise an issue for this ?

Reply:
> Yes, this information was not provided to you earlier (apologies), so thanks for making this change.
> 
> I think Nikhil's important reminder of the different ways the site serves pages (e.g. not just from the `pages` directory) was also something that could be missed, so maybe this type of site info should be in a reminder section in the `CONTRIBUTING.md` moving forward.
> 
> Cheers, Randy

Yes @randychilau. This would be very helpful for contributors. And since you have great documentation skills (as I, and many others might have noticed), it would be ideal that you advance on this task.

Reply:
> This seems like a good first issue for someone. Thoughts on this @leecalcote @Shivam-AfA. Should I raise an issue ?

@8rxn If the functionality of this PR depends on this change, it would be good that to take care of this ourselves. If not, then yes please, raise an issue.

Reply:
I meant the documentation, which I think Randy will now be taking care of. Rest of the everything will be Implemented by myself.

Reply:
Super helpful. Thank you, @randychilau